scp/scp.py

```
@@ -0,0 +1,226 @@
+                        # Split the message into chunks if it exceeds 2000 characters
+                        for i in range(0, len(message), 2000):
+                            await ctx.send(message[i:i + 2000])
+                    else:
+                        await ctx.send(f"No SCPs found in category: {category}.")
+                else:
+                    await ctx.send("Failed to fetch SCP articles. Please try again later.")
+
+    @scp_group.command(name='random')
+    async def random_scp(self, ctx):
+        """Fetch a random SCP article."""
+        random_number = random.randint(1, 9999)  # scp range
```

Ruff (0.8.2)

Undefined name `random`

(F821)